tools/log-collector/helper.go

```diff
@@ -13,6 +13,9 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 
+	// To authenticate kubeEnv on gke cluster
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
No you don't need to add this in every file.. You just need to remove the gcp from last.. This auth package has authentiction for gcp, azure and many more and we want to authenticate for all..
```
_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Reply:
Yes, made changes. My bad, completely forgot about `go mod tidy` and `vendor` last time